src/index.html

```diff
@@ -15,8 +15,101 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="/"
+        class="logo-link"
+      >
+        <img
+          src="./images/logo.png"
+          alt="MOYO Logo Header"
+        />
+      </a>
+      <nav>
+        <ul class="nav-list">
+          <li class="nav-list-link-item">
+            <a
+              href="/"
+              class="nav-list-link is-active"
```

Comment:
your class names are too long.
you need to use the main class for the main block, for example - `nav'.
For lower classes, you should display the very essence of the element (list, item, link, etc.) using 2 underscores
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a
              href="/"
              class="nav__link is-active"
```